Sources/GRPCNIOTransportHTTP2Posix/Config+TLS.swift

```diff
@@ -167,6 +168,14 @@ extension HTTP2ServerTransport.Posix.TransportSecurity {
     /// use at that point in time.
     public var certificateReloader: (any CertificateReloader)?
 
+    /// Override the certificate verification with a custom callback that must return the verified certificate chain on success.
+    /// Note: The callback is only used when `clientCertificateVerification` is *not* set to `noVerification`!
+    public var customVerificationCallback:
+      (
+        @Sendable ([NIOSSLCertificate], EventLoopPromise<NIOSSLVerificationResultWithMetadata>) ->
+          Void
+      )?
```

Comment:
From the Swift API design guidelines, https://github.com/apple/swift-nio/pull/3368, you should label closure parameters (auto-complete will suggest the names as variable names).

```suggestion
    public var customVerificationCallback: (
      @Sendable (
        _ certificates: [NIOSSLCertificate],
        _ promise: EventLoopPromise<NIOSSLVerificationResultWithMetadata>
      ) -> Void
    )?
```